app/wrappers/wakes/google_analytics_api_wrapper.rb

```diff
@@ -31,6 +65,66 @@ def credentials
     @credentials ||= Google::Auth::ServiceAccountCredentials.from_env(AUTHORIZATION_SCOPE)
   end
 
+  Page = Struct.new(:rows, :end?)
+
+  class PageViews
+    attr_reader :count, :url
+
+    def initialize(url, count)
+      @url = url
+      @count = count
+    end
+
+    def to_s
+      "#{url}: #{count}"
+    end
+  end
+
+  class Url
+    attr_reader :uri
+    delegate :query, :path, :to => :uri
+
+    def initialize(uri)
+      @uri = URI.parse(uri)
+    end
+
+    def to_s
+      uri.to_s
+    end
+
+    def params
+      @params ||= CGI.parse(query || '')
+    end
+
+    def lang
+      return @lang if defined?(@lang)
```

Comment:
no, I mean use memoist's `memoize`, which we use because it handles this: https://github.com/matthewrudy/memoist/blob/master/test/memoist_test.rb#L233